### ISSUE TITLE
Despite what Clarivate says, do we actually have the MEDLINE (PMID) for merged records in WoS?

### ISSUE THREAD

**Issue by peetucket** [sul-dlss/sul_pub#262]
Clarivate tells us that when a record from the WoS client is returned, if this result also has a PMID record, we will not get the PMID as a field in the result.  However, we believe there may be some records where a MEDLINE ID *is* included in the result.  If there is no PMID, we may need to make a secondary API call to determine which WoS records have a PMID.  If it is included, we don't need to make this extra call.  

Look at some results coming back from a WoS query to see if there is a pattern where the MEDLINE ID is included.

- [x] grab some example WoS IDs that also have MEDLINE IDs 
- [ ] ask Rob if we can expext this to be comprehensive

**Comment by darrenleeweber** [comment]
`rails c` explorations...

- get data for Russ Altman
```ruby
wos_client = WosClient.new(Settings.WOS.AUTH_CODE, :info);
wos_queries = WosQueries.new(wos_client);
altman = wos_queries.search_by_name('Altman, Russ');
altman.count
# => 464
```

- look for PMID identifiers in the records
```ruby
identifiers = altman.rec_nodes.map {|rec| [rec.search('UID').text, Nokogiri::XML(rec.to_xml).at('/REC/dynamic_data/cluster_related/identifiers')] }.compact
# returns nested Arrays [ [UID, identifiers], ... ]
```

**Comment by darrenleeweber** [comment]
In https://github.com/sul-dlss/sul_pub/issues/193, @peetucket found that `WOS:000081515000015` returns a PMID using the alternative API query.  Trying this particular record, we get:

```ruby
records = wos_queries.retrieve_by_id('WOS:000081515000015')
records.print
```

The data does not have a PMID in the identifiers:
```xml
        <identifiers>
          <identifier type='issn' value='0094-2405'/>
          <identifier type='xref_doi' value='10.1118/1.598623'/>
        </identifiers>
```

**Comment by darrenleeweber** [comment]
So, this analysis lends support for using the additional API noted in https://github.com/sul-dlss/sul_pub/issues/193.  Before I jump on that band wagon, we also want to know what PubMed data we were harvesting that _could_ now be in the WoS records already - e.g. the abstract is now in the WoS record (and do we have the rights to republish the abstract?).  If we still need supplemental data from PubMed (either because WoS doesn’t have it or the rights are too restrictive), I agree we need to move on the links-API and then continue to get the PubMed data (whether we do it sync or async).

The only hesitation that I have is whether the corresponding PubMed publication is already in the WoS record set but it has a `MEDLINE:{x}` UID.  In other words, we might not need to go to PubMed to get the record if it’s in the WoS MEDLINE database already.  I guess I don’t fully understand the relationship between the `WOS` and `MEDLINE` databases and whether we can identify a record in each one; I know there’s been some commentary about it already, but I’m not totally clear yet.  I don’t understand how/why a WOS record that somehow is also a PubMed record is not also a MEDLINE record.

**Comment by darrenleeweber** [comment]
Do _all_ the MEDLINE records have PMID?  Yes, looks like they do:
```ruby
medline_uids = uids.select {|uid| uid =~ /^MEDLINE:/ };
pmids_found = medline_uids.map {|uid| [ uid, identifiers[uid].search('identifier').any? {|id| id['type'] == 'pmid' } ]};
pmids_found.all? {|rec| rec[1] }
#=> true
```

**Comment by darrenleeweber** [comment]
Should we specifically identify a sample of known “merged” records and have them looked at here in this issue?  I don’t know how to find a sample of known “merged” records.  Is the `WOS:000081515000015` a known “merged” record?  (Even if it is, we should not make conclusions based on one record, we should have a sample of "merged" records.)

**Comment by darrenleeweber** [comment]
Done - examination of merged records can occur in #257
